lib/middleware/fetch-forecast.js

```diff
@@ -0,0 +1,17 @@
+const { getForecast } = require('../services/getForecast');
```

Comment:
i wouldn't make this middleware. Because, like your comment says, you need stuff from the database to fetch the weather, it should be a method on your model